students/FlyStriker/dom-practice/script.js

```diff
@@ -0,0 +1,18 @@
+document.getElementById("defaultOpen").click();
+
+function openInstrument(active, InstrumentName) {
+    let i, tab_content, tab_links;
+
+tab_content = document.getElementsByClassName("tab_content");
+  for (i = 0; i < tab_content.length; i++) {
+     tab_content[i].style.display = "none";
```

Comment:
Please do not change styles directly in JS. 
You can set up all styles in CSS files and than just play with adding/removing classes to the tag